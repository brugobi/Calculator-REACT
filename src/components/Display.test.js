import React from 'react';
import renderer from 'react-test-renderer';
import Display from './Display';

describe('Display Component', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<Display />);
    expect(tree).toMatchSnapshot();
  });
});
