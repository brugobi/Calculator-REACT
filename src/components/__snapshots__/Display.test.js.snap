// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component matches snapshot 1`] = `
<div
  className="display"
>
  0
</div>
`;
